src/server/handlers/hello.ts

```diff
@@ -0,0 +1,37 @@
+/*****
```

Comment:
I think we can drop whole `hello` endpoint - it was implemented only as an example...

Reply:
done